packages/eas-cli/src/build/ios/credentials.ts

```diff
@@ -16,20 +16,16 @@ export async function ensureIosCredentialsAsync(
     return;
   }
 
-  const { projectDir } = buildCtx.commandCtx;
-
-  const ctx = await createCredentialsContextAsync(projectDir, {
-    nonInteractive: buildCtx.commandCtx.nonInteractive,
+  const ctx = await createCredentialsContextAsync(buildCtx.projectDir, {
```

Comment:
nit pick: for code in build directory we should use variable name `ctx` for build context and for contexts value from different scopes we should use name with some prefix e.g. credentialsCtx